README.md

```diff
@@ -16,7 +16,7 @@ Browse to https://localhost:5443 to see the app.
 
 ## Dependencies
 
-**python 2.x** (tested with python 2.7.10)
+**python 3.x** (tested with python 3.9.1)
 
 **OpenSSL 1.0** to be able to do modern TLS versions. Python together with 0.9.x has a bug that makes it impossible to select protocol in the handshake, so it cannot connect to servers that have disabled SSLv2.
```

Comment:
I'm not sure if that is still true. I didn't have to change anything neither on my system nor when running from docker.

Reply:
Mac OS comes with a newer version nowadays